tools/kicad/kicad_rs/src/parser.rs

```diff
@@ -360,3 +362,41 @@ impl<T: AsRef<str>> SplitCharN for Option<T> {
         }
     }
 }
+
+fn unescape(s: &str) -> String {
```

Comment:
Is this a "custom" unescaping function, or is it a "conventional" like done for e.g. URL escaping?

Reply:
Its custom, I just cobbled it together since `kicad_parse_gen` messes up escaping so badly.